README.md

```diff
@@ -165,8 +164,10 @@ For more information on the structure of the YAML configuration file, please vis
 | ------------------------- | -------- | ----------- |
 | `acrUsername`             | No       | The username used to authenticate push requests to the provided Azure Container Registry. If not provided, an access token will be generated via "az acr login" and provided to "docker login" to authenticate the requests. |
 | `acrPassword`             | No       | The password used to authenticate push requests to the provided Azure Container Registry. If not provided, an access token will be generated via "az acr login" and provided to "docker login" to authenticate the requests. |
+| `registryUsername`             | No       | The username used to authenticate push requests to the provided container registry using the "docker login" action. |
+| `registryPassword`             | No       | The password used to authenticate push requests to the provided container registry usign the "docker login" action. |
 | `azureCredentials`        | No       | Azure credentials used by the `azure/login` action to authenticate Azure CLI requests if the user has not previously authenticated in the workflow calling this action. |
-| `imageToBuild`            | No       | The custom name of the image that is to be built, pushed to ACR and deployed to the Container App by this action. _Note_: this image name should include the ACR server; _e.g._, `<acr-name>.azurecr.io/<repo>:<tag>`. If this argument is not provided, a default image name will be constructed in the form `<acr-name>.azurecr.io/github-action/container-app:<github-run-id>.<github-run-attempt>` |
+| `imageToBuild`            | No       | The custom name of the image that is to be built, pushed to the container registry and deployed to the Container App by this action. _Note_: this image name should include the registry server; _e.g._, `<acr-name>.azurecr.io/<repo>:<tag>` or `<registryUrl>/<repo>:<tag>`. If this argument is not provided, a default image name will be constructed in the form `<acr-name>.azurecr.io/github-action/container-app:<github-run-id>.<github-run-attempt>` |

```

Comment:
Let's update the latter part of this text for the default image we use to replace `<acr-name>.azurecr.io` with `<registryUrl>`, just so it's clear we're using _some_ registry, whether or be ACR or the generic container registry provided.